flang/lib/Lower/Bridge.cpp

```diff
@@ -94,10 +94,11 @@ struct IncrementLoopInfo {
   template <typename T>
   explicit IncrementLoopInfo(Fortran::semantics::Symbol &sym, const T &lower,
                              const T &upper, const std::optional<T> &step,
-                             bool isUnordered = false)
+                             bool isConcurrent = false)
```

Comment:
unordered is also used for array operation. how is this handled now?

Reply:
This is done somewhere else though not here. In particular, `genImplicitLoops` in `ConvertExpr.cpp` where `fir.do_loop .... unordered` loops are still created. See: https://github.com/llvm/llvm-project/blob/main/flang/lib/Lower/ConvertExpr.cpp#L4393.